secretstores/local/env/envstore.go

```diff
@@ -39,15 +39,15 @@ func (s *envSecretStore) GetSecret(req secretstores.GetSecretRequest) (secretsto
 }
 
 // BulkGetSecret retrieves all secrets in the store and returns a map of decrypted string/string values
-func (s *envSecretStore) BulkGetSecret(req secretstores.BulkGetSecretRequest) (secretstores.GetSecretResponse, error) {
-	r := map[string]string{}
+func (s *envSecretStore) BulkGetSecret(req secretstores.BulkGetSecretRequest) (secretstores.BulkGetSecretResponse, error) {
+	r := map[string]map[string]string{}
 
 	for _, element := range os.Environ() {
 		envVariable := strings.Split(element, "=")
-		r[envVariable[0]] = envVariable[1]
+		r[envVariable[0]] = map[string]string{envVariable[0]: envVariable[1]}
```

Comment:
This seems like it has a bug for cases where an env-var value contains an `=`.  You could end up with a slice of more than 2 elements. This should use `SplitN`

Reply:
For an example, think about base64 encoded text that uses `=` as padding. A the trailing `=`s would be removed, and a strict parser would fail to decode it.

Reply:
Good one!